sale_order_variant_mgmt/models/sale_order.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    # This field is for avoiding conflicts with sale_stock module, that
+    # adds product_tmpl_id, and its possible modifications. This field name
+    # for sure won't conflict
+    product_tmpl_id_sale_order_variant_mgmt = fields.Many2one(
```

Comment:
Can you explain? The sale order line's product_tmpl_id is a related field on the line's product's template id, just like this one. Logic wise, it seems safe to assume the field is not customized further. What is the problem then with using this existing field?